src/components/InsightsData/index.js

```diff
@@ -0,0 +1,50 @@
+import RichTypography from "@commons-ui/core/RichTypography";
+import { Typography, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import InsightCard from "@/pesayetu/components/InsightCard";
+import Section from "@/pesayetu/components/Section";
+
+const InsightsData = ({ title, subtitle, items, ...props }) => {
+  let itemsData = [];
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+
```

Comment:
Let's use `Hidden`

Reply:
@KhadijaMahanga this is just to determine how many items to be rendered and not to render a different design.